src/main/java/org/jabref/logic/importer/fileformat/PdfContentImporter.java

```diff
@@ -254,6 +254,9 @@ private boolean isUnwantedText(TextPosition previousTextPosition, TextPosition t
             if (textPosition == null || previousTextPosition == null) {
                 return false;
             }
+            if (" ".equals(textPosition.getUnicode())) {
```

Comment:
I think, there could be multiple spaces etc.

```suggestion
            if (StringUtil.isBlank(textPosition.getUnicode())) {
```